src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr

```diff
@@ -8,10 +8,10 @@ vs.dependencies
   vs.dependency id=Microsoft.Net.4.6.1.FullRedist.Resources
 
 folder InstallDir:\MSBuild\15.0\Bin
-    file source=$(OutputPath)\VBCSCompiler.exe vs.file.ngen=yes
```

Comment:
`ngen=yes` means *use the default ngen setting for vs*.  Apparently they didn't what to have all that text in the option.